Dockerfile

```
@@ -0,0 +1,13 @@
+FROM golang:1.23
+
+WORKDIR /app
+
+COPY go.mod go.sum main.go parcel.go tracker.db ./
```

Если хочешь отдельно скачивать зависимости, то нужны только два файла
```suggestion
COPY go.mod go.sum ./
```